drivers/video/video_ctrls.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_VIDEO_CTRLS_H_
+#define ZEPHYR_INCLUDE_VIDEO_CTRLS_H_
+
+#include <zephyr/device.h>
+#include <zephyr/sys/slist.h>
+
+/*  Control flags */
+#define VIDEO_CTRL_FLAG_READ_ONLY  0x0004
+#define VIDEO_CTRL_FLAG_WRITE_ONLY 0x0040
+#define VIDEO_CTRL_FLAG_VOLATILE   0x0080
+
+/*  Control types */
+enum video_ctrl_type {
+	VIDEO_CTRL_TYPE_INTEGER = 1,
```

Comment:
This enums have to be documented

Reply:
I think the enums are already self-explanatory here, no ? The type of the control either an integer or an int64, etc.
Could you give some insights on what we should put here for documentation ?

Reply:
Have a look over here https://github.com/zephyrproject-rtos/zephyr/blob/baa49f6f32d5632c50c2c65dda44493dfe8ef8a3/include/zephyr/drivers/sensor.h#L61 #82377 

All the enums need to be documented even though it might be self-explanatory.  Also the enum declaration needs to have doxygen`@brief`.